lighthouse-viewer/app/src/main.js

```diff
@@ -43,3 +43,10 @@ Promise.all(loadPolyfillPromises).then(_ => {
   // eslint-disable-next-line no-new
   new LighthouseViewerReport();
 });
+
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('/sw.js', {scope: './'}).catch(e => {
```

Comment:
isn't that scope implied? (iirc, the default scope matches the path of the SW resource?)